pkg/util/jsonutil/seek_test.go

```diff
@@ -0,0 +1,74 @@
+// Copyright (C) 2017 ScyllaDB
+
+package jsonutil
+
+import (
+	"strings"
+	"testing"
+)
+
+const sample = `
+{
+	"name": "Kevin",
+	"cats": [
+		"Belle",
+		"Pedro",
+		"Cassidy",
+		"dogs"
+	],
+	"dogs": [
+		"Chloe",
+		"Ellie",
+		"Izzy",
+		"Jessie",
+		"Luna"
+	],
+	"fish": [
+		"Billy"
+	]
+}
+`
+
+type Person struct {
+	Name string
+	Dogs []string
+	Cats []string
+}
+
+func TestSeekAndDecode(t *testing.T) {
```

Comment:
Seek is too simplistic to be called seek tbh.
It would benefit from struct awareness. 
I'd suggest it accepts super simplified jsonpath stings i.e. `store/book/author` so that if one passes `foo` it's clear you mean the top level foo.